tools/csv-merger/csv-merger.go

```diff
@@ -478,13 +479,10 @@ func addRelatedImage(images []csvv1alpha1.RelatedImage, image string) []csvv1alp
 
 func panicOnError(err error, info ...string) {
 	if err != nil {
-		moreInfo := ""
-		if len(info) > 0 {
-			moreInfo = strings.Join(info, " ")
-		}
+		moreInfo := strings.Join(info, " ")
 
 		log.Println("Error!", err, moreInfo)
-		panic(err)
+		os.Exit(1)
```

Comment:
If we're no longer panicking on error, maybe it would be better to rename `panicOnError` to something like `exitOnErrror` ?
btw, with panic we're printing the stack trace for easier debug, and run deferred functions, and in os.Exit we're just terminating immediately. why we don't want to use panic any more?
also, in panic the exit code was 2, and here you're changing it to 1.